src/figure.js

```diff
@@ -0,0 +1,12 @@
+
+// Wrap the plot in a figure with a caption, if desired.
+export function figureWrap(svg, {width}, caption) {
+  if (caption == null) return svg;
+  const figure = document.createElement("figure");
+  figure.style = `max-width: ${width}px`;
```

Comment:
This appears to be sneaking in a fix for #356? I’m going to remove this; I’d rather address it separately.